src/test/java/api/requests/RequestsAPIRetrievalTests.java

```diff
@@ -54,14 +56,11 @@ public void canGetARequestById()
     ExecutionException,
     TimeoutException {
 
-    String enumeration = "DUMMY_ENUMERATION";
-
     UUID facultyGroupId = patronGroupsFixture.faculty().getId();
     UUID staffGroupId = patronGroupsFixture.staff().getId();
 
     final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet(
-      itemBuilder -> itemBuilder
-        .withEnumeration(enumeration)
+      itemBuilder -> itemsFixture.addCallNumberStringComponents().apply(itemBuilder)
         .withCopyNumbers(asList(ONE_COPY_NUMBER, TWO_COPY_NUMBER))
```

Comment:
Could this be refined so that it isn't mixing applying a function and calling the builder directly, I find this difficult to follow.

If the function `addCallNumberStringComponents` is to be used outside of the fixture, please could it be moved somewhere shared.